package.json

```diff
@@ -40,7 +40,8 @@
     "standard-version": "^4.4.0"
   },
   "peerDependencies": {
-    "vue": "^2.6.6"
+    "vue": ">=2.6"
```

Comment:
hats (`^`) are preferred. 

If this is your desired value, please write as `^2.6.0`

Reply:
Will revert that change. Thanks for catching that error.